db/seeds/partials/portfolio_transactions.rb

```diff
@@ -4,73 +4,88 @@
 
 if mike
   portfolio = Portfolio.find_or_create_by(user: mike)
-  stock = Stock.first
 
-  ######################################################################
-  # Create 4 completed orders and transactions for the Student user
-  ######################################################################
+  ######################################################################################
+  # Seed Mike's portfolio with initial deposit, purchases, sells, and cash transactions
+  ######################################################################################
+
+  # Deposit $15,000.00 into the portfolio as initial cash
```

Comment:
Can you please remove all the comments, the code is self documenting so we don't need these here.